mutator/expression/comparison.go

```diff
@@ -0,0 +1,47 @@
+package expression
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+
+	"github.com/zimmski/go-mutesting/mutator"
+)
+
+var tokenMutations = map[token.Token]token.Token{
+	token.LSS: token.LEQ,
+	token.LEQ: token.LSS,
+	token.GTR: token.GEQ,
+	token.GEQ: token.GTR,
+}
+
+func init() {
+	mutator.Register("expression/comparison", MutatorComparison)
+}
+
+// MutatorComparison implements a mutator to change comparisons.
+func MutatorComparison(pkg *types.Package, info *types.Info, node ast.Node) []mutator.Mutation {
+	n, ok := node.(*ast.BinaryExpr)
+
```

Comment:
Remove empy line because the check of the "ok" variable belongs to the assignment (in this case)